src/views/mfa-verify/WindowsHelloErrorMessageView.js

```diff
@@ -18,14 +18,22 @@ define([
 
     var template = '\
       <span class="icon error-24"></span>\
-      <h4><strong>{{i18n code="enroll.windowsHello.error.notWindows" bundle="login"}}</strong></h4>\
+      <h4><strong>{{{message}}}</strong></h4>\
```

Comment:
Sorry, I meant to add the comment to the code, like here:
https://github.com/okta/okta-signin-widget/blob/master/src/EnrollU2FController.js#L93